### PR TITLE
Incorrect day average temperature in daily forecast

### PR DESCRIPTION
**Pull request by aseriy**
String to float conversion when calculating average day temperature.

### CODE REVIEW

File: integTests/ForecastDailyTest.php [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+    public function testWindMetric()
+    {
+        $forecast = $this->owm->getWeatherForecast('Moscow', 'metric', 'ru', '', 9);
+
+        $this->assertEquals('Moscow', $forecast->city->name);
+        $this->assertEquals('RU', $forecast->city->country);
+        /*
+        This assertion fails right now because the Daily Forecase XML provides
+        the city ID as 'geobaseid' attribute, not as 'id' in Current Weather.
+        $this->assertEquals(524901, $forecast->city->id);
```
Comment (cmfcmf) [inline integTests/ForecastDailyTest.php]: Could you elaborate on this one? Does this mean there currently is an issue with the existing PHP code?

Comment (aseriy) [inline integTests/ForecastDailyTest.php]: I believe so. I think you're leveraging CurrentWeather class in the forecast code, so the assumption is that OWM sends the same XML for both /weather and /forecast/daily calls. However, I believe the XML looks different. In the case of /weather, the city ID is in "id" attribute" while in the case of /forecast/daily, it's in the "geobaseid" attribute. When inside the forecast, the CurrentWeather Object instantiated, it's looking for the "id" attributed but not finding it. This results in the city ID being blank.

**Comment by cmfcmf** [comment]
This was included in #71. Thank you!

File: integTests/ForecastDailyTest.php (hunk 1 above)
Comment (cmfcmf) [inline integTests/ForecastDailyTest.php]: You were right. This is another inconsistency in their api :disappointed: Fixed it though.

**Pull request closed without merging**